### ISSUE TITLE
Quasar or REPL is adding .0 to integer values from MongoDB

### ISSUE THREAD

**Issue by damonLL** [quasar-analytics/quasar#1884]
- Using Quasar 14.1.2, not the latest but relatively new. 
- MongoDB 3.2.9

Not sure if this is a REPL issue or Quasar issue, it may also affect other datasources.  Hopefully my problem is self explanatory.  See the following output from MongoDB shell:

```
host9772250243(mongod-3.2.9)[PRIMARY:rs] demo4> db.num.find()
{
  "_id": ObjectId("588f47f52d43fba47ec2ad4c"),
  "intnum": 123,
  "stringnum": "456"
}
{
  "_id": ObjectId("588f47ff2d43fba47ec2ad4d"),
  "intnum": 789,
  "stringnum": "999"
}
Fetched 2 record(s) in 1ms
```

Now see the output from Quasar:
```
💪 $ select * from num
MongoDB:
db.num.find().limit(11);


Query time: 0.3s
 intnum | stringnum |
--------|-----------|
  123.0 | 456       |
  789.0 | 999       |
```

Is this simply the way the REPL displays an integer value to distinguish itself from a String value?  Without the `.0` at the end, it would appear as the same exact value, regardless of data type.

**Comment by sellout** [comment]
So, the REPL distinguishes numbers by right-aligning them – you can see that `stringnum` is left-aligned. Also, if you wanted to be _sure_ about the types, you could use `format = precise`, then you would see JSON instead of a pretty-printed table.

I’m guessing the problem here is MongoDB. We add a `.0` to integer-valued decimal numbers to distinguish them from actual integers. My guess is that those values are stored in the collection as decimal values, but the Mongo console doesn’t do anything to distinguish them from integers syntactically. We make the distinction explicit.

tl;dr: I think those numbers are stored as decimals in the collection, and we make that clear where the Mongo console doesn’t. To fix it, wrap the values like `NumberInt(123)` (or `NumberLong`) when inserting.

**Comment by sellout** [comment]
Since these numbers are stored as floating-point in Mongo, this is behaving as expected. See the previous comments for ways to get the desired behavior.
